src/core/pusher.ts

```diff
@@ -180,18 +180,28 @@ export default class Pusher {
     }
   }
 
-  bind(event_name : string, callback : Function) : Pusher {
-    this.global_emitter.bind(event_name, callback);
+  bind(event_name : string, callback : Function, context? : any) : Pusher {
+    this.global_emitter.bind(event_name, callback, context);
     return this;
   }
 
-  unbind(event_name: string, callback: Function) : Pusher {
+  unbind(event_name? : string, callback? : Function, context? : any) : Pusher {
```

Comment:
I'm not sure I follow. All pusher's `unbind` function does is delegate to the `unbind` function of `EventsDispatcher`? I'm just allowing you to set the `context` in the same way you already can on the channel.